Source/WebCore/Modules/encryptedmedia/MediaKeySession.cpp

```diff
@@ -463,6 +463,11 @@ void MediaKeySession::update(const BufferSource& response, Ref<DeferredPromise>&
                         enqueueMessage(messageType, WTFMove(message->second));
                     }
                 }
+#if USE(OPENCDM)
```

Comment:
If this doesn't follow the spec'd algorithms, I don't think it belongs here. Please comment on why you're making the changes you're making, If it was missing from the implementation but the spec mandates it, you will have to include the commentary from the spec.